spring-data-neo4j/src/main/java/org/springframework/data/neo4j/repository/config/Neo4jMappingContextFactoryBean.java

```diff
@@ -52,16 +59,28 @@ public void setApplicationContext(ApplicationContext applicationContext) throws
 	}
 
 	/*
- * (non-Javadoc)
- * @see org.springframework.beans.factory.config.AbstractFactoryBean#createInstance()
- */
+	 * (non-Javadoc)
+	 * @see org.springframework.beans.factory.config.AbstractFactoryBean#createInstance()
+	 */
 	@Override
-	protected Neo4jMappingContext createInstance() throws Exception {
+	protected Neo4jMappingContext createInstance() {
 
 		SessionFactory sessionFactory = beanFactory.getBean(SessionFactory.class);
 		Neo4jMappingContext context = new Neo4jMappingContext(sessionFactory.metaData());
 		context.initialize();
 
+		ConversionService conversionService = null;
+		try {
+			conversionService = beanFactory.getBean(ConversionService.class);
+		} catch (NoSuchBeanDefinitionException e) {
+			LOG.debug("Unable to find a conversion service to use for entity instantiation, using none");
+		}
+		sessionFactory.setEntityInstantiator(getEntityInstantiator(context, conversionService));
```

Comment:
`getEntityInstantiator(context, conversionService)` may be called with `conversionService = null` - is this ok?